service/src/main/kotlin/fi/nokiankaupunki/evaka/message.config/MessageConfiguration.kt

```diff
@@ -64,7 +64,7 @@ internal class NokiaMessageProvider(private val messageSource: MessageSource) :
 
     override fun getDefaultDecisionAddress(lang: MessageLanguage): DecisionSendAddress = when (lang) {
         MessageLanguage.FI, MessageLanguage.SV -> DecisionSendAddress(
-            street = "Harjukatu 21",
+            street = "Harjukatu 23",
```

Comment:
Where does 23 come from? 🙂

![image](https://github.com/Tampere/trevaka/assets/29153474/4c9fa9a4-3bb6-4536-a435-c3e8fca41211)

Reply:
It's a typo, 23 is correct.